pkg/services/sqlstore/migrator/spanner_dialect.go

```diff
@@ -82,3 +108,224 @@ func (s *SpannerDialect) DropIndexSQL(tableName string, index *Index) string {
 func (s *SpannerDialect) DropTable(tableName string) string {
 	return fmt.Sprintf("DROP TABLE %s", s.Quote(tableName))
 }
+
+func (s *SpannerDialect) ColStringNoPk(col *Column) string {
+	sql := s.dialect.Quote(col.Name) + " "
+
+	sql += s.dialect.SQLType(col) + " "
+
+	if s.dialect.ShowCreateNull() {
+		if col.Nullable {
+			// Nothing. Columns are nullable by default.
+		} else {
+			sql += "NOT NULL "
+		}
+	}
```

Comment:
Suggestion: That's kind of weird to read to me. Perhaps something like that?
```suggestion
	if s.dialect.ShowCreateNull() && !col.Nullable {
		sql += "NOT NULL "
	}
```